vendor/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -392,6 +392,8 @@ message PodSandboxConfig {
     map<string, string> annotations = 7;
     // Optional configurations specific to Linux hosts.
     LinuxPodSandboxConfig linux = 8;
+    // Optional configurations specific to Windows hosts.
+    WindowsPodSandboxConfig windows = 9;
```

Comment:
How did you update the vendor folder? Doesn't this require a bump to the `go.mod`?

Reply:
I believe this hasn't been updated in awhile, this was likely just to get feedback. A new cri-api version would need to be pulled, however it seems like they usually update the k8s dependencies as a unit, so we'd need to update all of the k8s deps to an alpha version which I'm not sure would be agreed on.

Reply:
Yeah I wasn't sure how to do this with vendor unit k8s got updated, bit of a chicken and egg here